xep-0369.xml

```diff
@@ -62,8 +67,8 @@
   </revision>
 </header>
 <section1 topic='Introduction' anchor='intro'>
-  <p>Multi-User Chat (MUC) is a major application of XMPP that was developed in 2002 and standardized in &xep0045;. The Mediated Infromation eXchange (MIX) protocol defined here implements the same basic MUC patterns in a more flexible and extensible way in order to address requirements that have emerged since MUC was developed. MIX supports all of the core chatroom features that are familiar from MUC, such as discussion topics and invitations. Like MUC, it also defines a strong access control model, including the ability to kick and ban users, to name  administrators, and to require membership in order to participate in channels. MIX is intended as a medium term replacement for MUC.</p>
-  <p>MUC exists and works, so why replace it? There are several reasons:</p>
```

Comment:
Removed rhetorical question which may be confusing to non-native English speakers.

Also rephrased the first sentence so that we don't start immediately talking about MUC instead of MIX which confused me for a moment before I read further. It's well worth mentioning MUC, but it felt odd to start with discussion of something that wasn't the actual spec (though this really doesn't matter).